calculator/script.js

```diff
@@ -0,0 +1,28 @@
+window.onload = function() {
+
+    var calc = document.getElementById('calc');
+    var operations = {
+        "+": function(a, b) {
+            return a + b;
+        },
+        "-": function(a, b) {
+            return a - b;
+        },
+        "*": function(a, b) {
+            return a * b;
+        },
+        "/": function(a, b) {
+            return a / b;
+        },
+        "%": function(a, b) {
+            return Math.floor(a / b);
+        }
+    };
+
+    calc.onclick = function() {
+        var a = parseFloat(aInput.value);
+        var b = parseFloat(bInput.value);
+        var op = document.getElementById('operation').value;
```

Comment:
each click on button you call `getElementById`, it's possible to make a reference to element outside this callback like you did with `calc` variable